runtime/compiler/control/J9Options.cpp

```diff
@@ -1752,53 +1732,62 @@ J9::Options::fePreProcess(void * base)
          }
       else
          {
-            UDATA largePageSize = 0;
-            UDATA largePageFlags = 0;
-            // -Xlp<size>, attempt to use specified page size
-            if (lpSize > 0)
+
```

Comment:
Unfortunately, the diff is such that it's very hard to understand if it's just a simple formatting or some code was removed....

Reply:
@gita-omr I've left you comments on what changes I've made. Please take a look and feel free to ask any further questions if it's not clear.